mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.kt

```diff
@@ -261,27 +256,12 @@ abstract class ContentController protected constructor(private val activity: Mai
     }
 
     fun showWebViewUi(ui: WebViewUi, isStackRoot: Boolean) {
-        val prefs = activity.getPrefs()
-        val activeServerId = prefs.getActiveServerId()
-        val title = if (prefs.getConfiguredServerIds().size <= 1 || activity.connection is DemoConnection) {
-            activity.getString(ui.titleRes)
-        } else {
-            val activeServerName = ServerConfiguration.load(prefs, activity.getSecretPrefs(), activeServerId)?.name
-            activity.getString(ui.multiServerTitleRes, activeServerName)
-        }
-
-        val webViewFragment = WebViewFragment.newInstance(
-            title,
-            ui.errorRes,
-            ui.urlToLoad,
-            ui.urlForError,
-            activeServerId,
-            isStackRoot,
-            ui.shortcutAction,
-            title,
-            ui.shortcutIconRes
+        val webViewFragment = ui.fragment.newInstance()
+        webViewFragment.init(
+            activity,
+            this,
+            isStackRoot
```

Comment:
This doesn't work. Fragments must be reinstatiable only from fragment state (e.g. after screen rotation), in which case that init() method isn't called. The arguments must be inferred from life cycle methods (e.g. onAttach) or the argument bundle.

Reply:
Ok, I removed `isStackRoot` and `activity` for now and I'll do the same with `callback` later.